src/webview/service/analyzeSettingsService.ts

```diff
@@ -35,8 +41,12 @@ export interface AnalyzeSettingsProps {
 export default class AnalyzeSettingsService {
     public readonly WAVEFORM_CANVAS_WIDTH = 1000;
     public readonly WAVEFORM_CANVAS_HEIGHT = 200;
+    public static WAVEFORM_CANVAS_VERTICAL_SCALE_MAX = 2.0;
+    public static WAVEFORM_CANVAS_VERTICAL_SCALE_MIN = 0.3;
     public readonly SPECTROGRAM_CANVAS_WIDTH = 1800;
     public readonly SPECTROGRAM_CANVAS_HEIGHT = 600;
+    public static SPECTROGRAM_CANVAS_VERTICAL_SCALE_MAX = 2.0;
+    public static SPECTROGRAM_CANVAS_VERTICAL_SCALE_MIN = 0.2;
```

Comment:
Is the reason why WAVEFORM_CANVAS_VERTICAL_SCALE_MIN = 0.3 is different from SPECTROGRAM_CANVAS_VERTICAL_SCALE_MIN = 0.2 to align the minimum canvas size?
Since this is adjusted by the user himself, is it possible to change both to 0.2?
If you really want to set different values (0.2 and 0.3), it may be better to comment the reason in the README.